src/supervisor.sbvr

```diff
@@ -0,0 +1,461 @@
+
+Term: api secret
+	Concept Type: Short Text (Type)
+
+Term: commit
+	Concept Type: Short Text (Type)
+
+Term: application
+
+	Fact type: application has slug
+	-- Not currently provided to Supervisor, optional, but would be nice for LocalUI
+		Necessity: each application has exactly one slug
+		Necessity: each slug is of exactly one application
+
+	Fact type: application has api port
+		Necessity: each application that has a slug that is equal to "balena-os/balena-supervisor", has exactly one api port that is equal to 48484.
+
+	Fact type: application has name
+	-- Not currently provided to Supervisor, optional, but would be nice for LocalUI
+		Necessity: each application has exactly one name.
+		Necessity: each application has an app name that has a Length (Type) that is greater than or equal to 4 and is less than or equal to 100.
+
+	Fact type: application has uuid
+	-- Optional: wait for v3 target state?
+		Necessity: each application has exactly one uuid.
+		Necessity: each uuid is of exactly one application.
+		Necessity: each application has a uuid that has a Length (Type) that is equal to 32.
+
+	Fact type: application is host
+
+	Fact type: application is healthy
+	-- GET /v1/healthy
+		Necessity: each application that has a slug that is equal to "balena-os/balena-supervisor" is healthy or not healthy.
+		-- Not sure about syntax / validity for "healthy" boolean
+	
+	Fact type: application has download progress
+	-- Maybe should be computed term of all image download progresses?
+		Necessity: each application has at most one download progress.
+
+	Fact type: application should be running release
+		Synonymous Form: release should be running on application
+		Necessity: each application should be running at most one release.
+
+	Fact type: application has version
+		Necessity: each application that has a slug that is equal to "balena-os/balena-supervisor" has exactly one version.
+
+	Fact type: application has api secret
+		Necessity: each application that has a slug that is equal to "balena-os/balena-supervisor" has at least one api secret.
+
+	Fact type: device belongs to application
+		Synonymous Form: application owns device
+		Necessity: each device belongs to at least one application.
+		-- Changed from "at most one application" to "at least" for multi-app -- valid?
+
+	Fact type: application has commit
+		Necessity: each application has exactly one commit.
+		Necessity: each commit belongs to exactly one application.
+		Necessity: each commit has a uuid that has a Length (Type) that is equal to 32.
+
+
+
+-- CONFIG / ENV VARS
+Term: value
+	Concept Type: Text (Type)
+
+Term: config var
+	Concept Type: Short Text (Type)
+
+	Fact type: config var has value
+		Necessity: each config var has exactly one value.
+
+	Fact type: application owns config var
+		Synonymous Form: config var belongs to application
+		Necessity: each application that has a config var must have a slug equal to "balena-os/balena-supervisor".
```

Comment:
I would like to see if we can get rid of config vars at least for the supervisor for now

Reply:
Removed the config var section